.github/workflows/acceptance-tests.yml

```diff
@@ -1,31 +1,58 @@
 name: Acceptance Tests
 
 on:
-  schedule:
-    - cron: "30 7 * * *"
-  workflow_dispatch:
+  issue_comment:
+    types: [created]
+
 
 concurrency:
   group: ci-${{ github.ref }}
   cancel-in-progress: true
 
 jobs:
   acceptance_tests:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
+    if: ${{ github.event.issue.pull_request && github.event.comment.body == '/acceptance_tests'}}
```

Comment:
I guess this this to trigger it only if the created comment is exactly:
`/acceptance_tests`